doc/doxydown/doxydown.pl

```diff
@@ -271,12 +271,12 @@ sub print_markdown {
             }
         }
 
-        if ($m->{'stucts'}) {
-            if ( scalar(@{ $m->{'stucts'} }) > 0 ) {
-                print "\n## Stucts\n\nThe module `$mname` defines the following stucts.\n\n";
+        if ($m->{'structs'}) {
+            if ( scalar(@{ $m->{'structs'} }) > 0 ) {
+                print "\n## Structs\n\nThe module `$mname` defines the following structs.\n\n";
```

Comment:
?

Reply:
I have no ideas why this has not been discovered previously - that's all I can say :)